packages/amplify-cli/src/commands/gen2-migration/refactor/resolvers/cfn-output-resolver.ts

```diff
@@ -161,6 +161,11 @@ class CfnOutputResolver {
             return {
               Arn: `arn:aws:lambda:${this.region}:${this.accountId}:function:${resourceIdentifier}`,
             };
+          case 'AWS::Kinesis::Stream':
+            return {
+              // output is already in ARN format
+              Arn: resourceIdentifier,
+            };
           default:
```

Comment:
Should probably default to resourceIdentifier instead of `undefined`... but we will take a deeper look at this later.